subprojects/docs/src/docs/release/notes.md

```diff
@@ -6,6 +6,21 @@ Here are the new features introduced in this Gradle release.
 IMPORTANT: if this is a patch release, ensure that a prominent link is included in the foreword to all releases of the same minor stream.
 Add-->
 
+### Timeouts for HTTP requests
+
+Previous versions of Gradle did not define a timeout for any HTTP requests. Under certain conditions e.g. network problems, unresponsive or overloaded servers this behavior could lead to hanging connections.
+Gradle now defines connection and socket timeouts for all HTTP requests. Please see [Configuring HTTP timeout](userguide/dependency_management.html#sec:http_timeout) for more details.
```

Comment:
:o: I'd be more explicitly here: Maybe say with HTTP and HTTPS requests.

Reply:
Done.